### PR TITLE
fix(Case Management): re-gen service and add examples

### PR DESCRIPTION
**Pull request by pyrooka**
## PR summary

This PR adds examples to the Case Management service.

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type  

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] New tests
- [ ] Build/CI related changes
- [ ] Documentation content changes
- [ ] Other (please describe)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

### CODE REVIEW

File: examples/test_case_management_v1_examples.py [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+
+            response = case_management_service.download_file(
+                case_number=case_number,
+                file_id=attachment_id,
+            )
+
+            file = response.get_result()
+
+            print('Attachment content-type:', response.get_headers()['content-type'])
+            print('Attachment contents:', file)
```
Comment (padamstx) [inline examples/test_case_management_v1_examples.py:318]: We'll need to do something slightly different to print out the contents of the downloaded "file".  When I run the examples, i get this output:
```
Attachment content-type: application/octet-stream
Attachment contents: <Response [200]>
```

**Review by padamstx — CHANGES_REQUESTED** [review]
Looks good!   Just one comment about how to print out the result of "download_file".

File: examples/test_case_management_v1_examples.py (hunk 1 above)
Comment (pyrooka) [reply examples/test_case_management_v1_examples.py]: You are totally right. I forgot to check the verbose logs. With the modification I have just pushed it looks like this:
`Attachment contents: b'This is the content of the file to upload.`
WDYT?

**Review by padamstx — APPROVED** [review]
LGTM

**Review by padamstx — APPROVED** [review]
LGTM

**Pull request merged**